mscore/musescore.cpp

```diff
@@ -246,6 +252,9 @@ const std::list<const char*> MuseScore::_basicNoteInputMenuEntries {
             "",
             "pad-rest",
             "",
+            "pad-raise",
+            "pad-lower",
+            "",
```

Comment:
I was not the first person to put an issue in the issue tracker requesting this. I already linked to another one, and I think there are more.

Is there a way that user can decide exactly which buttons are shown (rather than just choosing basic or advanced)?